frontend/src/pages/Traces/TraceLogs.tsx

```diff
@@ -35,38 +35,27 @@ export const TraceLogs: React.FC = () => {
 	const { projectId } = useProjectId()
 	const [query, setQuery] = useState('')
 
-	const {
-		logEdges,
-		loading,
-		error,
-		loadingAfter,
-		fetchMoreForward,
-		fetchMoreBackward,
-	} = useGetLogs({
-		query,
-		project_id: projectId,
-		logCursor: undefined,
-		startDate,
-		endDate,
-	})
+	const { logEdges, loading, error, loadingAfter, fetchMoreForward } =
+		useGetLogs({
+			query,
+			project_id: projectId,
+			logCursor: undefined,
+			startDate,
+			endDate,
+		})
 
 	const fetchMoreWhenScrolled = React.useCallback(
-		(
-			containerRefElement?: HTMLDivElement | null,
-			disableBackwards?: boolean,
-		) => {
+		(containerRefElement?: HTMLDivElement | null) => {
 			if (containerRefElement) {
 				const { scrollHeight, scrollTop, clientHeight } =
 					containerRefElement
 
 				if (scrollHeight - scrollTop - clientHeight < 100) {
 					fetchMoreForward()
-				} else if (!disableBackwards && scrollTop === 0) {
```

Comment:
Removed this behavior for logs because it resulted in extra unused requests, and we favor the load more via the "X new logs" button

Reply:
Maybe worth discussing if/when we would want to have this UX in the future. If we are going to rely on the "X more traces/logs" feature instead of this, maybe we want to just remove the `fetchMoreBackward` methods (and not add it in `useGetTraces`)?

Reply:
I agree, right now this doesn't seem to be behaving as expected and was creating tech debt on some of the infinite loading side. So definitely worth a discussion on the behavior here